generators/entity-server/templates/src/test/java/package/web/rest/EntityResourceIT.java.ejs

```diff
@@ -271,22 +272,27 @@ _%>
 
     private static final Integer <%=defaultValueName %> = <%= defaultValue %>;
     private static final Integer <%=updatedValueName %> = <%= updatedValue %>;
+    private static final Integer <%=smallerValueName %> = <%= defaultValue %> - 1;
```

Comment:
If I'm not misunderstanding what this PR does, these new `SMALLER_` fields will only be used if the JPA filtering is enabled.

So it would be better to add them conditionally (`<%_ if (jpaMetamodelFiltering) { ... }`) otherwise they will be dead/unused code in the generated application.